pipeline/api/__init__.py

```diff
@@ -47,7 +43,8 @@ def authenticate(token: str, url: str = PIPELINE_API_URL):
     PIPELINE_API_TOKEN = token
     global PIPELINE_API_URL
     PIPELINE_API_URL = url
-    # TODO: Change this url to an actual auth one, not status which just shows if the API is alive.
+    # TODO: Change this url to an actual auth one,
+    # not status which just shows if the API is alive.
```

Comment:
I'd say this comment can be removed; `GET /v2/users/me` _is_ an endpoint which requires authentication. I think this comment is a remnant from when the check hit `GET /v2/status`, which does not require authentication.